packages/indicators/src/types.ts

```diff
@@ -21,50 +21,58 @@ export interface AnalyticCluster {
   dataValues: Record<Analytic['dataElement'], Analytic['value']>;
 }
 
-type TypeFields = Record<string, string | number | Record<string, unknown>>;
+type DbValue = string | number | boolean | null | DbValue[] | { [key: string]: DbValue };
```

Comment:
Some intense type nesting here, I'm impressed TypeScript is happy to evaluate that!

Reply:
Yeah TypeScript is very sophisticated!